client/src/lib/user/state/state.ts

```diff
@@ -88,6 +98,43 @@ export const getCompletedSessionByIdSelector: GetCompletedSessionByIdSelector =
     }
   };
```

Comment:
Refactor to a hook

Reply:
Also add a comment about not using selectors but `getCurrentUserStateSelector` is an off-case